CMakeLists.txt

```diff
@@ -14,7 +13,16 @@ enable_testing()
 include(SetupGTest)
 
 # C++ version
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+if ("${CMAKE_VERSION}" LESS "3.1")
```

Comment:
We want to still support 14.04 until it's EOL (per @barulicm), which I think is a good idea. There are still a lot of 14.04 machines that might want to run our software :P.